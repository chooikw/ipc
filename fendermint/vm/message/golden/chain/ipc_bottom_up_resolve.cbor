a163497063a16f426f74746f6d55705265736f6c7665a2676d657373616765a6676d657373616765a2676d657373616765a4697375626e65745f6964a264726f6f741b9d90a88ccd78fb44686368696c6472656e80666865696768741a9b585af8756e6578745f76616c696461746f725f7365745f69641b021efbfd76c28d4c72626f74746f6d5f75705f6d65737361676573d82a58460001db3cf2353f8a6ce023d82764f292f87fd04cc2895f54f3cbcbad4973ef516dec044566d6240f5af0af5bae3f103d39ce4aea873e98c0dd4e6a096ca0e45fb9e8fa6f96db6b6365727469666963617465a16a7369676e61747572657381a26976616c696461746f72550298ff23d2658fc164b71c00aaf69cc80944904799697369676e61747572654302f3006772656c61796572550100081504f5d04bd6ff93e7d657890570ff681fbd6873657175656e63651be5eeeefeb598fbdd696761735f6c696d69741b64332e1f1a675b6f6b6761735f6665655f6361705100ab6db406751786574bd9f9d5df6d148d6b6761735f7072656d69756d40697369676e61747572654901f42ab93b6f4a6c81