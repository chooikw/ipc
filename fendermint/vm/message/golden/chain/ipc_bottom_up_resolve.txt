Ipc(BottomUpResolve(SignedRelayedMessage { message: RelayedMessage { message: CertifiedMessage { message: BottomUpCheckpoint { subnet_id: SubnetID { root: 11353759983297166148, children: [] }, height: 2606258936, next_validator_set_id: 152836503414934860, bottom_up_messages: Cid(bahntz4rvh6fgzybd3atwj4us7b75atgcrfpvj46lzowus47pkfw6ybcfm3lcid226cxvxlr7ca6tttsk5kdt5gga3vhguclmudsf7opi7jxznwy) }, certificate: MultiSig { signatures: [ValidatorSignature { validator: Address("f2td7shutfr7awjny4acvpnhgibfcjar4zvj3qsza"), signature: Signature { sig_type: BLS, bytes: [243, 0] } }] } }, relayer: Address("f1aaebkbhv2bf5n74t47lfpcifod7wqh55n4yoy4i"), sequence: 16568442856878439389, gas_limit: 7220165338726882159, gas_fee_cap: TokenAmount(227867598982136728714.712790251088974989), gas_premium: TokenAmount(0.0) }, signature: Signature { sig_type: Secp256k1, bytes: [244, 42, 185, 59, 111, 74, 108, 129] } }))